### PR TITLE
Set timezone and  limit for dependabot PR

### PR DESCRIPTION
**Pull request by shivam-sharma7**
Set timezone and  limit for dependabot PR

[Provide a detailed explanation of the solution you are proposing, including any relevant code snippets or screenshots.]

## Changes Made

- [List the changes you made in this pull request, including any new features or bug fixes.]

## How to Test

[Provide instructions on how to test the changes you made, including any relevant details like configuration steps or data to be used for testing.]

## Screenshots

[Include screenshots, if relevant, to help reviewers understand the changes you made.]

## Notes

[Any additional notes or information that you would like to share with the reviewers.]

## Checklist

- [ ] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [ ] Changes have been thoroughly tested
- [ ] <a href="https://github.com/tailwarden/docs.komiser.io">Documentation</a> has been updated, if necessary
- [ ] Any dependencies have been added to the project, if necessary

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,9 +4,13 @@ updates:
...
       interval: "weekly"
+      time: "23:00"
+      timezone: Europe/London
       labels:
-      - "npm dependencies"
+      - "dependencies"
+    open-pull-requests-limit: 10
       reviewers:
       - "mlabouardy"
       - "ShubhamPalriwala"
```
Comment (AvineshTripathi) [inline .github/dependabot.yml:14]:
```suggestion
      - "Azanul"
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -4,9 +4,13 @@ updates:
     directory: "/dashboard"  
     schedule:
       interval: "weekly"
+      time: "23:00"
+      timezone: Europe/London
       labels:
-      - "npm dependencies"
+      - "dependencies"
+    open-pull-requests-limit: 10
```
Comment (AvineshTripathi) [inline .github/dependabot.yml:11]:
```suggestion
    open-pull-requests-limit: 5
```

10 is bit too high @Azanul wdyt?

**Review by AvineshTripathi — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (shivam-sharma7) [reply .github/dependabot.yml:14]: @AvineshTripathi you mean remove ShubhamPalriwala as reviewers ?

Comment (AvineshTripathi) [reply .github/dependabot.yml:14]: yes he is a bit busy these days so we dont want to flood his mail with notifications

File: .github/dependabot.yml (hunk 2 above)
Comment (AvineshTripathi) [reply .github/dependabot.yml:11]: Nvm I think 10 sounds reasonable for now, we can change those when needed later on

Comment (shivam-sharma7) [reply .github/dependabot.yml:11]: I want to tell you why 10? because we dependent on more than 10 node dependencies.

**Review by AvineshTripathi — APPROVED** [review]

**Review by Azanul — APPROVED** [review]

**Pull request merged**